scripts/zones/Northern_San_dOria/npcs/Aurege.lua

```diff
@@ -41,6 +41,7 @@ function onEventFinish(player, csid, option)
     if csid == 521 and player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER) == QUEST_AVAILABLE then
         player:addQuest(SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER)
     elseif csid == 516 then
-        npcUtil.completeQuest(player, SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER, {ki = tpz.ki.MAP_OF_KING_RANPERRES_TOMB, title = tpz.title.DAYBREAK_GAMBLER, xp = 2000})
+        npcUtil.completeQuest(player, SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER, {ki = tpz.ki.MAP_OF_KING_RANPERRES_TOMB, title = tpz.title.DAYBREAK_GAMBLER, xp = 2000 * EXP_RATE})
+        player:messageSpecial(ID.text.KEYITEM_OBTAINED, tpz.ki.MAP_OF_KING_RANPERRES_TOMB) -- Map of King Ranperre's Tomb
```

Comment:
npcUtil isn't displaying the message because [the table key it expects inside the rewards table is `keyItem` instead of `ki`](https://github.com/project-topaz/topaz/blob/release/scripts/globals/npc_util.lua#L348-L350)

Reply:
@ibm2431 https://github.com/project-topaz/topaz/blob/release/scripts/globals/keyitems.lua#L3011

`tpz.ki` is equivalent to `tpz.keyItem`

Reply:
@Xephera looks like npcUtil.completeQuests() already has exp rate baked in:
https://github.com/project-topaz/topaz/blob/8985d2af68876367592222836db1f8e5b15c8175/scripts/globals/npc_util.lua#L372

Reply:
The _enums_ are equivalent, but the table that npcUtil.completeQuest takes is not~

Reply:
Ah, I misunderstood table, thinking ki ***is***  a table (because our "enums" _are_  tables) and totally missed that the argument input here is its _own_  table.

Reply:
Yeah, it's confusing. Wondering if npcUtil functions should be tweaked to accept both "ki" and "keyItem" as field names.

Reply:
I fixed the exp_rate thing but it doesn't give a key item notification regardless for whatever reason so I'm leaving that part in until we know why.

Reply:
`npcUtil.completeQuest(player, SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER, {keyItem = tpz.ki.MAP_OF_KING_RANPERRES_TOMB, title = tpz.title.DAYBREAK_GAMBLER, xp = 2000 * EXP_RATE})` didn't work?

Reply:
Just gonna second that
```
player:messageSpecial(ID.text.KEYITEM_OBTAINED, tpz.ki.MAP_OF_KING_RANPERRES_TOMB) -- Map of King Ranperre's Tomb
```
shouldn't be needed if you change
```
npcUtil.completeQuest(player, SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER, {ki = tpz.ki.MAP_OF_KING_RANPERRES_TOMB, title = tpz.title.DAYBREAK_GAMBLER, xp = 2000})
```
to
```
npcUtil.completeQuest(player, SANDORIA, tpz.quest.id.sandoria.EXIT_THE_GAMBLER, {keyItem = tpz.ki.MAP_OF_KING_RANPERRES_TOMB, title = tpz.title.DAYBREAK_GAMBLER, xp = 2000})
```
notice the change from `ki = tpz.ki.MAP_OF_KING_RANPERRES_TOMB` to `keyItem = tpz.ki.MAP_OF_KING_RANPERRES_TOMB`